docs/guides/dynamo_run.md

```diff
@@ -273,7 +273,7 @@ cd target/debug
 ./dynamo-run in=http out=sglang --model-path ~/llms/DeepSeek-R1-Distill-Llama-70B/ --tensor-parallel-size 8
 ```
 
-To pass extra arguments to the sglang engine see *Extra engine arguments* below.
+To pass extra arguments to the sglang engine see [Extra engine arguments](#Extra engine arguments) below.
```

Comment:
Does it work like that? I suspect you need the dashes: `#extra-engine-arguments`

e.g. https://github.com/ai-dynamo/dynamo/blob/main/docs/guides/dynamo_run.md#extra-engine-arguments